### DISCUSSION TITLE
docker+machine

### DISCUSSION THREAD

**Discussion by marcoveretti** [npalm/terraform-aws-gitlab-runner#1126] [General]
Sorry: new to this module.
Worker starts correctly but I got this error "Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?"
I've enabled the ssm on workers and if I log into worker itself I got this warning using **ssm-user**.
Of course using **root** user docker works.
What Am I Missing in runner configuration ?

`  runner_worker = {
    type       = "docker+machine"
    ssm_access = true
  }
`

**Comment by kayman-mk** [comment]
Welcome!

Did you check #1025 and #779?

**Reply by marcoveretti** [reply]
Sorry: I'll check asap

**Comment by marcoveretti** [comment]
Thanks.
This works for me 
`  runner_worker_docker_options = {
    privileged = "true"
    volumes      = ["/cache", "/var/run/docker.sock:/var/run/docker.sock"]
  }
`
